src/Export/EntityManager.php

```diff
@@ -197,6 +197,7 @@ public function queueExportLocalEntityAllSyncs(
     $syncs = $this->configManager->getSyncs([
       'local_entity' => [
         'type_id' => $entity->getEntityTypeId(),
+        'bundle' => $entity->bundle(),
```

Comment:
**CHANGE REQUEST**

There may still be synchronizations that do not define a bundle; either because the entity does not have bundles, or because the synchronization should apply to all bundles. The current code with the new filter will mistakenly filter out those synchronizations.

I think we can improve the `getSyncs` method to receive the following:
```
$this->configManager->getSyncs([
  'bundle' => [
    'id' => $entity->bundle,
    'optional' => TRUE,
  ],
]);
```

And then in the code, if we are given `'optional' => FALSE` (let's make this the default) then the synchronization needs to define a bundle and be the requested one (`$entity->bundle`), if `'optional' => TRUE` then the synchronization does not need to define a bundle but if it does it have to be the requested one.